le_dash/rollcall.py

```diff
@@ -17,6 +17,19 @@ def lecture(mpid, sort_key='student.last_name'):
     return sorted(attendance, key=attrgetter(sort_key))
 
 
+def episodes_by_series():
+    resp = episode_lookup()
+    result = []
+    for r in resp:
+        try:
+            result.append(
```

Comment:
I think `r.title` is what you want here?

Also, the formatting of the options is a little fugly. Maybe it could be "Title (series: [series id])"